code/game/objects/items/devices/last_shelter.dm

```diff
@@ -16,7 +16,7 @@ GLOBAL_DATUM(last_shelter, /obj/item/device/last_shelter)
 
 /obj/item/device/last_shelter/New()
 	..()
-//	GLOB.last_shelter = src
+	GLOB.last_shelter = src
```

Comment:
// Occulus Edit

Reply:
Not strictly an occulus edit considering this is the original code, which just had been commented out.

Reply:
Git will still see a conflict if this code is changed by upstream, and we update.

Reply:
Well, I end up having to resolve all sorts of conflicts related to faction objectives more often than not when we update, so this still wouldn't change anything in the long run!